packages/plugin-trpc/web/src/lib/handler.ts

```diff
@@ -0,0 +1,40 @@
+import type { Handle } from '@sveltejs/kit';
+import type { Dict } from '@trpc/server';
+import { resolveHTTPResponse } from '@trpc/server/http';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const createTrpcHandler = (endpointURL: string, router: any): Handle => {
```

Comment:
the type for router needs to be dynamic, based on the model that's created. I don't have a solution for this just yet, that's why it's set to `any`

Reply:
Maybe that's a manual step required, or we can find a solution, once we have the generation of the router automated 🤔

Reply:
`any` is fine for the time being, we an review before we merge the PR to see if there's a feasible solution.

Reply:
Found a solution for it, we may use a generic with type `Router extends AnyRouter` here